src/main/java/core/basesyntax/report/ReportDataImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.report;
+
+import core.basesyntax.DataBase;
+import java.util.Map;
+
+public class ReportDataImpl implements ReportData {
+    private static final int TITLE_INDEX = 0;
+    private static final String TITLE = "fruit,quantity";
+
+    @Override
+    public String[] createDataReport() {
```

Comment:
```suggestion
    public String createDataReport() {
```
давай відразу формувати звіт